### PR TITLE
 Add/Updated FXIOS-7508 [v120] Localized String

### PR DESCRIPTION
**Pull request by jnrahme**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7508)

## :bulb: Description
Updated strings to match renders in figma 

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -1005,19 +1005,19 @@ extension String {
                 value: "Welcome to an independent internet",
                 comment: "String used to describes the title of what Firefox is on the welcome onboarding page for current version in our Onboarding screens.")
             public static let Description = MZLocalizedString(
-                key: "Onboarding.Welcome.Description.v114",
+                key: "Onboarding.Welcome.Description.v120",
```
Comment (yoanarios) [inline Client/Frontend/Strings.swift:1008]: Updating strings is complicated I'm not sure if we should follow these steps from our [wiki](https://github.com/mozilla-mobile/firefox-ios/wiki/How-to-add-and-modify-Strings#steps-1) for these cases. The string freeze is October 17 so we still have time let me investigate how it should be done

Comment (jnrahme) [reply Client/Frontend/Strings.swift:1008]: Do you mean to add the NotUsed part? I can add it

Comment (jnrahme) [reply Client/Frontend/Strings.swift:1008]: Its been updated @yoanarios pls let me know if I need to do any more updates

**Review by lmarceau — APPROVED** [review]

**Review by yoanarios — APPROVED** [review]

**Comment by jnrahme** [comment]
Made a new PR

**Pull request closed without merging**